### DISCUSSION TITLE
Hide/expand sidebar

### DISCUSSION THREAD

**Discussion by kevisong** [grafana/grafana#37243] [Dashboards, Panels & Visualizations]
Is it possible to hide / expand left sidebar? so that the dashboard can go full screen

'Cycle view mode' can achieve this but variables is not selectable in 'Cycle view mode'

![image](https://user-images.githubusercontent.com/39701004/127118602-df2fb292-f1f9-482a-a358-d3998017e7e7.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
